app/models/invitation.server.ts

```diff
@@ -34,6 +34,16 @@ export async function getInvitation(id: Invitation["id"] ){
   })
 }
 
+export async function updateInvitation(id: Invitation["id"], status: string, guests: number ){
```

Comment:
Not a biggie by any means but it would be slightly more clear if we did `status: Invitation['status']` and `guests: Invitation['guests']`